src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-overview-dashboard/rgw-overview-dashboard.component.ts

```diff
@@ -83,24 +79,23 @@ export class RgwOverviewDashboardComponent implements OnInit, OnDestroy {
 
   ngOnInit() {
     this.interval = this.refreshIntervalService.intervalData$.subscribe(() => {
-      this.daemonSub = this.rgwDaemonService.list().subscribe((data: any) => {
-        this.rgwDaemonCount = data.length;
-      });
-      this.HealthSub = this.healthService.getClusterCapacity().subscribe((data: any) => {
-        this.objectCount = data['total_objects'];
-        this.totalPoolUsedBytes = data['total_pool_bytes_used'];
-        this.averageObjectSize = data['average_object_size'];
-      });
-      setTimeout(() => {
+      this.fetchDataSub = combineLatest([
+        this.rgwDaemonService.list(),
+        this.rgwBucketService.fetchAndTransformBuckets(),
+        this.rgwBucketService.totalNumObjects$,
+        this.rgwBucketService.totalUsedCapacity$,
+        this.rgwBucketService.averageObjectSize$,
+        this.rgwBucketService.getTotalBucketsAndUsersLength()
+      ]).subscribe(([daemonData, _, objectCount, usedCapacity, averageSize, bucketData]) => {
```

Comment:
if we are not using `this.rgwBucketService.fetchAndTransformBuckets()` result anywhere, then can we avoid calling it?